### DISCUSSION TITLE
[A-L] (13 Febbraio 2024) Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#407] [Esami A-L]
Consideriamo la seguente relazione $\Large \prec$ definita su coppie di intervalli aperti della retta reale:

### Fila A

<div align="center">

$\Large (x, y) \prec (w, z)$ se e solo se $\Large((x, y) \neq (w, z)$ e $\Large(x, y) \subseteq (w, z))$ oppure $\Large y < w$.

</div>

Indicare se le seguenti affermazioni sono vere o false.

1. $\Large(-1, \pi) \prec (0, \frac{1}{2}\ )$.

2. La relazione $\Large \prec$ è riflessiva.

3. La relazione $\Large \prec$ è simmetrica.

4. La relazione $\Large \prec$ è transitiva.

### Fila B

<div align="center">

$\Large (x, y) \prec (w, z)$ se e solo se $\Large (x, y) \subseteq (w, z)$ oppure $\Large y \leq w$.

</div>

Indicare se le seguenti affermazioni sono vere o false.

1. $\Large (−1, \pi) \prec (0, \pi)$.

2. La relazione $\Large \prec$ non è riflessiva.

3. La relazione $\Large \prec$ non è simmetrica.

4. La relazione $\Large \prec$ non è transitiva.

### Entrambe le file

5. L’inversa della relazione $\Large \prec$ è simmetrica.

**Comment by FeddyLix17** [comment]
### Fila A

1. falso, poichè $\Large (-1, \pi) \not \subseteq (0, \frac{1}{2}\ )$ e $\Large \pi \not < 0$

2. falso, poichè $\Large ((a, a) = (a,a))$ e $\Large a \not < a$

3. falso, poichè se $\Large y < w$ allora  $\Large w \not < y$

4. vera

### Fila B

1. falso

2. falso, la condizione $\Large (x, y) \subseteq (x, y)$ sarà sempre vera

3. vero

4. falso

### Entrambe le file

5. non essendo entrambe le relazioni simmetriche, non lo saranno neanche le loro inverse, quindi l'affermazione è falsa per entrambe le file

**Reply by BenjiKunji** [reply]
La 2 della fila B dovrebbe essere falsa.

**Reply by DavideGalilei** [reply]
La 2B è sbagliata: $\large{(1, 2) \subseteq (1, 2)}$ è soddisfacibile, quindi la risposta è **Falso**, la relazione è riflessiva.
Nella fila B non c'è il requisito che $\large{(x, y) \subsetneq (w, z)}$

**Reply by cek753** [reply]
La 4A dovrebbe essere falsa e la 4B vera
4A:
![IMG_0046](https://github.com/user-attachments/assets/65bd61e9-3a2c-40b8-9f08-d56c7f162ff9)
4B:
![IMG_0047](https://github.com/user-attachments/assets/2270ceba-4602-4cb3-b7a2-44ba3809cab1)
